apis/nucleus/src/components/NebulaApp.jsx

```diff
@@ -110,6 +115,10 @@ export default function boot({ app, context }) {
       context(ctx) {
         (async () => {
           await rendered;
+          // Should be done here, unify contraints and interactions
+          if (ctx) {
+            unifyContraintsAndInteractions(ctx);
```

Comment:
Is it necessary to do this here, or will it be triggered on line 68 above anyway, making this superfluous? I can understand if this is needed though, to ensure it's kicks in immediately...

Reply:
The one on 68 should only run once (on boot), this one is for when someone updates the context by running embed.context(...)